.dockerignore

```diff
@@ -0,0 +1,13 @@
+.github
+.pytest_cache
+.venv
+.vscode
+.env
+.env.j2
+.env.template
+.gitignore
+ansible-inventory
+ansible-playbook.yml
+geckodriver.log
+README.md
+todoapp.service
```

Comment:
You could also ignore your Dockerfile itself, since this otherwise seems pretty thorough and it doesn't need to be inside the image. Or only copy over `todo_app` and `gunicorn.conf.py` in your Dockerfile.